bindings/swc_cli/Cargo.toml

```diff
@@ -26,6 +26,7 @@ rayon = "1"
 relative-path = "1.6.1"
 serde = { version = "1", features = ["derive"] }
 serde_json = { version = "1", features = ["unbounded_depth"] }
+sourcemap = "6.2.2"
```

Comment:
Please revert this dep upgrade

Reply:
As mentioned in https://github.com/swc-project/swc/pull/6973#issue-1594003697, we need the source_root feature that was added to the [sourcemap](https://crates.io/crates/sourcemap) crate in [v6.2.0](https://github.com/getsentry/rust-sourcemap/releases/tag/6.2.0).

Reply:
The sourcemap crate is only used to serialize/deserialize the source map. It is not used to generate a new one.

Reply:
They are used. That's how the cargo work

Reply:
But well, I think this would be fine